native/packages/react-native-bpk-component-icon/src/BpkIcon.js

```diff
@@ -42,7 +44,14 @@ const styles = StyleSheet.create({
 const mapCharacterCode = characterCode =>
   String.fromCharCode(parseInt(characterCode, 16));
 
-const BpkIcon = props => {
+type IconName = $Keys<typeof iconMappings>;
+type Props = {
+  icon: IconName,
+  small: ?boolean,
```

Comment:
This shouldn't be optional, we always expect it to have a valid value (true or false).

Reply:
But it *is* an optional proptype - that's what this is analogous to, right?

Reply:
No that’s not true. The prop type is external to the component, but flow is internal i.e while a consumer can omit `small`, inside the component we expect it to always have a value. In this case React and the default props ensure that that is the case

Reply:
Ok, makes sense. I'll change it